fhir-server/src/main/java/com/ibm/fhir/server/FHIRApplication.java

```diff
@@ -61,7 +63,9 @@ public FHIRApplication() {
                 classes.add(Search.class);
                 classes.add(Update.class);
                 classes.add(VRead.class);
-                classes.add(WellKnown.class);
+                if (FHIRConfigHelper.getBooleanProperty(FHIRConfiguration.PROPERTY_SECURITY_OAUTH_SMART_ENABLED, false)) {
+                    classes.add(WellKnown.class);
```

Comment:
confirmation, if OAuth doesn't exist, we don't add and advertise?

Reply:
this properly lives UNDER oauth, so yes, if oauth field is missing, this smart field will be missing too (and thus false).

however, it *is* possible to have the fhir-server-config oauth field present and set to false, while the underlying smart_enabled field is set to true.
In this case, we will advertise the wellknown endpoint, but we won't advertise the oauth endpoints in the CapabilityStatement.  I considered changing this if to check that BOTH are enabled, but decided that this behavior would probably be acceptable (and maybe even useful in rare circumstances).